apps/web/src/app/[locale]/aot-2023/_components/fem-promo.tsx

```diff
@@ -1,8 +1,16 @@
+import Image from 'next/image';
 import { cn } from '@repo/ui/cn';
 import { Sparkle } from '@repo/ui/icons';
-import Image from 'next/image';
```

Comment:
can you delete this component and the i18n keys

Reply:
all thing about FEM ?

Reply:
yep, its going to be removed end of december

Reply:
DONE !